.github/workflows/publish-pages.yml

```diff
@@ -0,0 +1,37 @@
+name: Publish LinkChecker Documentation on GitHub Pages
+
+on:
+    push:
+      branches: ["master"]
+
+jobs:
+  run:
```

Comment:
The indentation seems inconsistent, but who's cares?

(Me.  I care.  I"m cursed with care for irrelevant detail, while letting important things slide.  Also, I have yamllint integrated with my text editor of choice.  2-space indents for YAML files look nice.)

Reply:
All good, another top tip. yamllint errors and warnings (except document start and truthy for on:) fixed.
Also updated documentation.md for the automation.

Reply:
It's beautiful now.

:sheds tear: